### PR TITLE
Adjust Margins on Tutorial Modal

### PR DESCRIPTION
**Pull request by wgranger**
Close issue #2100 

Default left margin caused the tutorial modal to be pushed too far to the right on small screens. Increasing the top margin pushed the modal down far enough to reveal the close button.

### CODE REVIEW

**Comment by srallen** [comment]
A few things: 
- Consider using `em` or [viewport units](https://developer.mozilla.org/en-US/docs/Web/CSS/length) instead of fixed pixel sizes. Either are better for responsiveness. Also, for consistency with our other styles.
- `!important` should be generally avoided and I also don't think it's necessary in this case. The tutorial.styl is further down the cascade.
- Also remember our alphabetic ordering of style properties convention.
- Consider a media query since this is only a issue for small viewports.

**Comment by wgranger** [comment]
Good call on the styling adjustments. Need to get in the habit of keeping things alphabetical.

File: css/tutorial.styl [hunk 1]
```diff
@@ -3,6 +3,10 @@
   max-width: 400px
   padding: 0 0 1.5em
 
+  @media screen and (max-width: 400px)
+    margin-left: 0vh
```
Comment (srallen) [inline css/tutorial.styl]: `vh` is viewport height, so it doesn't make sense to use `vh` here. To be consistent with our other styles, `em` would be best. From MDN:

> The CSS properties line-height, font-size, margin-bottom and margin-top often has a value expressed in em.

File: css/tutorial.styl [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+    margin-top: 5vh
```
Comment (srallen) [inline css/tutorial.styl]: To be consistent with our other styles, `em` would be best. From MDN:

> The CSS properties line-height, font-size, margin-bottom and margin-top often has a value expressed in em.

**Pull request merged**